### DISCUSSION TITLE
Compared with HTTP Vue loader, vue3 SFC loader is too large. I hope the author can release a lightweight version as large as HTTP Vue loader

### DISCUSSION THREAD

**Discussion by click33** [FranckFreiburger/vue3-sfc-loader#98] [Ideas]
Compared with HTTP Vue loader, vue3 SFC loader is too large. I hope the author can release a lightweight version as large as HTTP Vue loader

**Comment by FranckFreiburger** [comment]
have a look at https://github.com/FranckFreiburger/vue3-sfc-loader/blob/main/docs/faq.md#why-vue3-sfc-loader-is-so-big

**Reply by click33** [reply]
Thank you for your answer
